python/lsst/daf/butler/cli/cmd/commands.py

```diff
@@ -149,6 +153,147 @@ def prune_collection(**kwargs):
     script.pruneCollection(**kwargs)
 
 
+pruneDatasets_wouldRemoveMsg = "The following datasets would be removed:"
+pruneDatasets_willRemoveMsg = "The following datasets will be removed:"
+pruneDatasets_askContinueMsg = "Continue?"
+pruneDatasets_didRemoveAforementioned = "The datasets were removed."
+pruneDatasets_didNotRemoveAforementioned = "Did not remove the datasets."
+pruneDatasets_didRemoveMsg = "Removed the following datasets:"
+pruneDatasets_noDatasetsFound = "Did not find any datasets."
+pruneDatasets_errPurgeAndDisassociate = unwrap(
+    """"--disassociate and --purge may not be used together: --disassociate purges from just the passed TAGged
+    collections, but --purge forces disassociation from all of them. """
```

Comment:
`TAGGED` is actually the `CollectionType` enum value, so I'd capitalize the whole thing.

I'd also rephrase to "--disassociate removes from", just to keep the verb "purge" as implying "removal from a RUN collection."